build.gradle

```diff
@@ -69,6 +69,9 @@ dependencies {
     testRuntimeOnly group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: jUnitVersion
 
     compile "org.mockito:mockito-core:2.+"
+    implementation "de.jensd:fontawesomefx-fontawesome:4.7.0-11"
```

Comment:
Hey just wanted to check if this additional dependency on an external library is approved? Was this mentioned on the Slack channel?

Reply:
Actually what are the approved external libraries?

Reply:
Yeah, I'm not sure about this either. I just remember our tutor mentioning that whenever we use external libraries we should get it approved by asking on Slack first

Reply:
ill ask in lecture tmrw

Reply:
oh noo, ok i'll check in lecture tomorrow.